### DISCUSSION TITLE
Discord Bearer Client - Login to Discord with a Bearer Token??

### DISCUSSION THREAD

**Discussion by CE1CECL** [aiko-chan-ai/DiscordBotClient#116] [Q&A]
Ok, I am very smart about discord bots, ever since my Discord Bot was created in 2020, and my selfbot that was PoS before then, but somehow am not with Bearer tokens and webhook access? What are these? Can you "login" to a webhook? I only know they may not have gateway access.

**Comment by CE1CECL** [comment]
https://discord.com/developers/docs/reference#authentication-example-bearer-token-authorization-header

**Reply by aiko-chan-ai** [reply]
bearer tokens and webhook tokens cant view channels, they only have the function of sending messages (webhook) or oauth2 (bearer)
